apps/dashboard/components/logs/queries/queries-status-row.tsx

```
@@ -0,0 +1,27 @@
+export const StatusRow = ({ status }: StatusRowProps) => {
+  const shownStatus = status && status.length <= 4 ? status : status?.slice(0, 3);
+  const overflowStatus = status && status.length > 4 ? status.slice(3) : null;
+  return shownStatus ? (
+    <div className="flex flex-row items-center justify-start w-full gap-2">
+      <div className="flex-col font-mono font-normal text-xs text-gray-9 align-start w-[43px]">
+        Status
+      </div>
+      <ChartActivity2 className="size-3.5 mb-[2px]" />
+      <span className="font-mono text-xs font-normal text-gray-9">{shownStatus[0]?.operator}</span>
+      {shownStatus.map((item) => {
+        return <QueriesPill value={item.value} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)